#include "internal_exception.hpp"

InternalException::InternalException(InternalStatus status_) :
	status(status_)
{}
